### PR TITLE
check latest beta version before bump new beta version

### PR DESCRIPTION
**Pull request by gunhappy**
In case if your registry has a beta version for example `1.0.0-beta.0` but version in your dev branch is `1.0.0` then if version-pong bump a beta version it will cause error because version `1.0.0-beta.0` is already exist in your registry.

So, it should check your latest beta version in registry before bumping.

### CODE REVIEW

File: src/update/bumpBetaVersion/index.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+import isEmpty from 'lodash/isEmpty'
+import Spinner from 'ora'
+import semver from 'semver'
+import standardVersion from 'standard-version'
+
+async function bumpBetaVersion(prefixObj?: Record<string, any>) {
+  const bumpBetaSpinner = Spinner('Updating package.json...')
+  bumpBetaSpinner.start()
+
+  const packageJson = JSON.parse(fs.readFileSync('package.json').toString())
```
Comment (Thanadej8) [inline src/update/bumpBetaVersion/index.ts]: This command is duplicate in other files.you should be extract method

File: src/update/bumpBetaVersion/index.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    const { stdout: version } = await execa.command(
+      `npm view ${packageName}@beta version --registry=${packageJson.publishConfig.registry}`,
+    )
+    versionOnRegistry = version
+  } else {
+    const { stdout: version } = await execa.command(
+      `npm view ${packageName}@beta version`,
+    )
+    versionOnRegistry = version
+  }
```
Comment (Thanadej8) [inline src/update/bumpBetaVersion/index.ts]:
```
const publishWithRegistryCommand = `npm view ${packageName}@beta version --registry=${packageJson.publishConfig.registry}`
const publishCommand = `npm view ${packageName}@beta version`
const { stdout: version } = await execa.command(
      packageJson.publishConfig?.registry ? publishWithRegistryCommand : publishCommand
    )
```

File: src/update/bumpBetaVersion/index.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    if (isEqualLatestVersion) {
+      packageJson.version = versionOnRegistry
+    }
+  }
+
+  const packageJsonString = JSON.stringify(packageJson, null, 2)
+  fs.writeFileSync('package.json', packageJsonString)
+
+  await standardVersion({
+    prerelease: 'beta',
```
Comment (Thanadej8) [inline src/update/bumpBetaVersion/index.ts]: PublishLevel.BETA

**Comment by Thanadej8** [comment]
lgtm

**Review by ReiiYuki — APPROVED** [review]

**Pull request merged**